Code/Editor/Undo/Undo.cpp

```diff
@@ -335,7 +342,6 @@ void CUndoManager::Redo(int numSteps)
     {
         GetIEditor()->UpdateViews(eUpdateObjects);
     }
-    GetIEditor()->GetLogFile()->FormatLine("Redo (Undo:%d,Redo:%d)", m_undoStack.size(), m_redoStack.size());
     GetIEditor()->GetObjectManager()->InvalidateVisibleList();
```

Comment:
Move the log to the above while loop. If `numSteps` is N (>1), it should output for N times.
Also, by putting it here, no assertion is needed for to check the stack size before using node in it.